package.json

```diff
@@ -18,8 +18,10 @@
     "update-staging": "yarn upgrade @carbon/ibmdotcom-react@latest && yarn upgrade @carbon/ibmdotcom-styles@latest"
   },
   "dependencies": {
-    "@carbon/ibmdotcom-react": "^1.8.0",
-    "@carbon/ibmdotcom-styles": "^1.8.0",
+    "@carbon/ibmdotcom-react": "^1.9.0-alpha.4534",
+    "@carbon/ibmdotcom-services": "^1.9.0-alpha.4519",
```

Comment:
Not sure why services and utilities are added here?

Reply:
Gotcha, because I ran `yarn update` and this command updates the 4 packages (react, styles, services and utilities), I hadn't seen. Thanks to notice, I will remove the services and utilities from `yarn update`, ok? @jeffchew